src/pages/orgtest.astro

```diff
@@ -0,0 +1,44 @@
+---
+import PastProjectsCard from "../components/PastProjectsCard.astro";
+
+const orgID = "1"
```

Comment:
Instead of hardcoding these pages, you can use dynamic routing to generate all these pages at once. 
Refer to the `Dynamic Paths` and `SSG Mode` in the [Astro Documentation here](https://docs.astro.build/en/core-concepts/routing/#dynamic-routes). Basically we want to create all the pages for us, by supplying what all possible organization IDs can be there. Try using this.

Reply:
removed the page